src/lib/plugins/moderation.js

```diff
@@ -119,7 +119,9 @@ module.exports.server = function (serv, settings) {
     },
     action (params) {
       params = params.split(' ')
-      const player = serv.getPlayer(params[0])
+      const [inputUsername] = params.split(' ')
```

Comment:
this assumes inputUsername was found and is a string, maybe you can check before so it won't throw if the result of .split is null?

Reply:
params is already split, why did you write another split?